packages/reactivity/src/reactive.ts

```diff
@@ -40,7 +40,7 @@ const canObserve = (value: any): boolean => {
   )
 }
 
-export function reactive<T extends object>(target: T): UnwrapNestedRefs<T>
```

Comment:
This change is unnecessary. `reactive(ref(0))` and `readonly(ref(0))` should return the ref, not the unwrapped number.

Reply:
done